bootstrap.sh

```diff
@@ -139,6 +139,24 @@ function install_grpc() {
 install_dep "gRPC" "1.10.0" "$VTROOT/dist/grpc" install_grpc
 
 
+# Install protoc.
+function install_protoc() {
```

Comment:
Do you know why `python -m grpc_tools.protoc` didn't work for you for the Go gRPC plugin?

The last time I changed this, I intentionally removed the "protoc" binary as dependency because it already comes with the Python gRPC installation if you launch it as Python module.

If I remember correctly, the current code in the master branch should work both on MacOSX and Linux.

If you let me know which error you hit, let's try to fix that and avoid the "protoc" dependency. Alternatively, we need to better document in this CL that there are two "protoc" in play.

Regardless of that, I am fine with getting rid of the temporary files for the Go gRPC plugin.

Reply:
I don't recall the error I hit trying to make grpc_tools work. I'll try to reproduce it later tonight so we can either fix the error or fail to fix it and document reasons for having two copies of protoc.

Reply:
Was this MacOSX or Linux?

Reply:
Don't remember. I think I tried OS X first and then Linux, but I didn't spend very long since it seemed easier to build something simpler that worked than debug something more complicated that wasn't working (the duplicate copy of protoc wasn't something I was thinking about).

Reply:
When I naively use `$(PROTOC_COMMAND)` instead of `path/to/protoc` on OS X, my command is hanging forever without printing any errors.